src/main/markdown/release-notes.md

```diff
@@ -59,10 +59,71 @@ The GWT Release Notes
 
 <a id="Release_Notes_Current"></a>
 
-<!-- ## <a id="Release_Notes_2_8_0_RC1"></a> Release Notes for 2.8.0 (RC1)
+## <a id="Release_Notes_2_8_0_RC1"></a> Release Notes for 2.8.0 (RC1)
+
+### Highlights
+- Support for several Java 8 API added.
+- Fix memory leak with Java 8 compilation.
+- Source level set to Java 8 by default
 
 ### Deprecations
-- Classic dev mode deprecated.You can switch back to dev mode for your test cases temporarily by passing gwt.args='-devMode'. -->
+- Classic dev mode deprecated.You can switch back to dev mode for your test cases temporarily by passing gwt.args='-devMode'.
+- Deprecate c.g.g.validation
```

Comment:
@ManfredTremmel The main reasons for deprecation are:

  - there is no official maintainer for it and
  - its still Validation API 1.0 while people constantly ask for Validation API 1.1 compatibility, either because of class path issues (they have one class path for client and server code and get a version conflict if server uses Validation API 1.1) or they simply want to use Validation API 1.1 on the client (but of course new things like method validation won't work because of missing reflection)

The team knows that the Validation 1.0 code generally works and thats why it will be moved to a separate project instead of simply being deleted. I think the overall goal is to disentangle some GWT features from Bean Validation (Editor Framework / Request Factory) and then let Bean Validation become a 3rd party project.

However if you want to become a maintainer of Bean Validation then maybe the team / steering group is also fine with keeping the code inside GWT. But that is something better discussed in gwt-contrib: https://groups.google.com/forum/#!forum/google-web-toolkit-contributors